layouts/partials/blm-styles.hbs

```diff
@@ -0,0 +1,122 @@
+<style>
```

Comment:
this could be moved to scss, but honestly it is way easier to integrate as a handlebar partial.